portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,6 +25,7 @@
 public class DataServlet extends HttpServlet {
   //Default value "Comment" for testing purposes
   private String commentDatastoreKey = "Comment";
+
```

Comment:
Did you mean to add this?

Reply:
Yeah, the style checker was telling me to separate the method def from the block above it with a newline